config/webpack/webpack.dev.config.js

```diff
@@ -0,0 +1,43 @@
+var path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './index.ts',
+  output: {
+    path: path.resolve(__dirname, '..', '..', 'dist', 'bundles'),
+    filename: 'help-client.umd.js',
+    library: 'BBHelpClient',
+    libraryTarget: 'umd'
+  },
+  devtool: 'inline-source-map',
+  resolve: {
+    extensions: ['.ts', '.js']
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        loader: 'ts-loader'
+      },
+      {
+        test: /\.scss$/,
+        loaders: ['style-loader', 'css-loader?localIdentName=[hash:base64:10]', 'sass-loader'],
+        exclude: '/node_modules/'
+      },
+    ]
+  },
+  devServer: {
```

Comment:
Why do you need a dev server?

Reply:
Dev server is useful for making changes to both the widget and the SPA that the widget loads. The communication between the two needs a server in order to have a "secure/authorized" communication with the SPA to client.  This helps us not have to refresh and re-load every time we make changes while developing the client.